app/models/intake.rb

```diff
@@ -270,4 +270,23 @@ def get_or_create_spouse_auth_token
     update(spouse_auth_token: new_token)
     new_token
   end
+
+  def mixpanel_data
+    dependents_under_6 = dependents.any? { |dependent| dependent.age_at_end_of_tax_year < 6 }
+    had_earned_income = had_a_job? || had_wages_yes? || had_self_employment_income?
```

Comment:
Should this be `had_self_employment_income_yes?` like `had_wages_yes?`

Reply:
oops good catch